LoRaEngine/LoraKeysManagerFacade/IoTCentralImp/Definitions/DeviceTemplateInfo.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoraKeysManagerFacade.IoTCentralImp.Definitions
+{
+
+    internal class DeviceTemplateInfo
```

Comment:
Is there a reason for this being `internal` and other types being `public` in this namespace?

Reply:
fixed